kubeflow/openvino/openvino.libsonnet

```diff
@@ -0,0 +1,101 @@
+{
+  local k = import "k.libsonnet",
+  local util = import "kubeflow/core/util.libsonnet",
+  local service = k.core.v1.service,
+  local deployment = k.apps.v1beta1.deployment,
+  local container = deployment.mixin.spec.template.spec.containersType,
+
+  new(_env, _params):: {
+    local params = _env + _params {
+      labels: {
+        role: _params.name,
+      },
+      imageURL: _params.registry + "/" + _params.repoPath + "/" + _params.image,
+    },
+    params:: params,
+
+    local ovService =
+      service.new(
+        name=params.name,
+        selector=params.labels,
+        ports=service.mixin.spec.portsType.newNamed("ov", params.servicePort, params.targetPort),
+      ).withType(params.serviceType) +
+      service.mixin.metadata.
+        withNamespace(params.namespace).
+        withAnnotationsMixin({
+        "getambassador.io/config":
+          std.join("\n", [
+            "---",
+            "apiVersion: ambassador/v0",
+            "kind:  Mapping",
+            "name: ov-mapping-" + params.name + "-get",
+            "prefix: / " + params.name + "/",
+            "rewrite: /",
+            "method: GET",
+            "service: " + params.name + "." + params.namespace + ":" + params.servicePort,
+          ]),
+      }),
+    ovService:: ovService,
+
+    local ovContainer =
+      container.new(
+        params.name, params.imageURL
+      ).withImagePullPolicy("IfNotPresent").
+        withArgs([
+        "ie_serving",
+        "model",
+        "--model_path",
+        "/opt/ml/resnet",
```

Comment:
shouldn't model path be a param?
also the model name below.

Reply:
added these as params